packages/imperative/CHANGELOG.md

```diff
@@ -422,6 +426,10 @@ All notable changes to the Imperative package will be documented in this file.
 
 - Major: First major version bump for V3
 
+## `5.26.1`
+
+- Bugfix: Export new Proxy class from Zowe imperative package. [#2205](https://github.com/zowe/zowe-cli/pull/2205)
```

Comment:
we should explain why this is a bug. right now it reads like an enhancement.

Reply:
How about:
> Added missing export for `Proxy` class to Imperative package.

Reply:
@t1m0thyj -- perfecto!